src/main/java/org/opentripplanner/routing/algorithm/mapping/RaptorPathToItineraryMapper.java

```diff
@@ -119,7 +116,7 @@ else if (pathLeg.isTransferLeg()) {
         Itinerary itinerary = new Itinerary(legs);
 
         // Map general itinerary fields
-        itinerary.generalizedCost = path.generalizedCost();
+        itinerary.generalizedCost = path.otpDomainCost();
         itinerary.arrivedAtDestinationWithRentedBicycle = mapped != null && mapped.arrivedAtDestinationWithRentedBicycle;
 
         if(optimizedPath != null) {
```

Comment:
The line below maps `waitTimeAdjustedGeneralizedCost` - should that also be converted to be an `otpDomainCost`?

```java
            itinerary.waitTimeAdjustedGeneralizedCost = optimizedPath.getWaitTimeOptimizedCost();
```

Reply:
Yes, I guess that would make sense. I will make an update.